### PR TITLE
updated FAQ with module 2 qestions

### PR DESCRIPTION
**Pull request by RNAer**

### CODE REVIEW

**Comment by wasade** [comment]
@adamrp @ElDeveloper, this looks good yesterday except for the FAQ ordering which should be fixed (changes don't want to come down on webdev right now...). If this looks good to you guys, could we get this live?

**Comment by ElDeveloper** [comment]
Ok, reading all the points right now, so far it looks great!

File: www/americangut/FAQ.psp [hunk 1]
```diff
@@ -180,7 +196,101 @@
...
+                            <p>We should also mention that since we are only interested in your microbes, we do not sequence human genomic DNA in our typical analyses. Where it is possible for human DNA to be sequenced (e.g., the Beyond Bacteria kits), we remove the human DNA using the same bioinformatics approaches undertaken in the NIH-funded Human Microbiome Project and approved by NIH bioethicists. Additionally, there is so little human DNA in fecal, skin and mucus samples that the chances of us being able to sequence your entire human genome are almost none, even if we tried. </p>
+                        </div>
+                        <div class="separator"></div>
+
+                        <h3 class="faq_header" id="faq17">Where can I find more detailed information about my sample?</h3>
+                        <div class="faq_content">
+                            <p>You can find it on our website. Log in with your account name and password to download the full dataset.</p>
```
Comment (ElDeveloper) [inline www/americangut/FAQ.psp]: I would vouch for re-phrasing this, as users cannot download the "full dataset" but instead a print with preliminary visualizations.

File: www/americangut/FAQ.psp [hunk 2]
```diff
@@ -180,7 +196,101 @@
...
+                        <div class="faq_content">
+                            <p>Although we still don’t have a predictable way to change the gut microbiome in terms of increasing or decreasing the abundances of specific bacteria, we do know that a variety of factors influence gut microbial community composition. Diet is a major factor affecting the gut microbiome so by changing your diet, you may be able to affect your gut microbiome. We still don’t fully understand probiotics but know that they can influence your gut microbiome while you are actively taking them. Factors such as stress can also influence the gut microbiome. However, it is important to remember that there are factors we can’t change, such as age or genetics, that can affect the gut microbiome.</p>
+                        </div>
+                        <div class="separator"></div>
+
+                        <h3 class="faq_header" id="faq20">Can I get another copy of my results?</h3>
+                        <div class="faq_content">
+                            <p>You can download a copy from our website. Log in with your account name and password.</p>
```
Comment (ElDeveloper) [inline www/americangut/FAQ.psp]: `account name` -> `Kit ID`. And perhaps expand on the instructions, something like "go to the left side bar, human samples > participant name > SAMPLE NUMBER > SAMPLE NUMBER.pdf".

**Comment by ElDeveloper** [comment]
Aside from these minor comments I think it should be good to go.

**Comment by RNAer** [comment]
i can change with what Yoshiki commented, if everyone agrees

**Comment by ElDeveloper** [comment]
@adamrp @wasade

**Comment by RNAer** [comment]
updated, and live on webdev too.

**Comment by ElDeveloper** [comment]
If people could take a look that would be fantastic:

http://webdev.microbio.me/americangut/FAQ.psp

We need this branch merged ASAP to point participants to this information.

**Comment by wasade** [comment]
I'm happy

**Comment by adamrp** [comment]
Reviewing now...

**Comment by ElDeveloper** [comment]
Thanks!

File: www/americangut/FAQ.psp [hunk 3]
```diff
@@ -180,7 +196,101 @@
...
+
+                        <h3 class="faq_header" id="faq20">Can I get another copy of my results?</h3>
+                        <div class="faq_content">
+                            <p>You can download a copy from our website. Log in with your account name and password, go to the left side bar, move your mouse to Human Samples -> PARTICIPANT NAME -> SAMPLE NUMBER, and then click on SAMPLE NUMBER.pdf to download it.</p>
+                        </div>
+                        <div class="separator"></div>
+
+                        <h3 class="faq_header" id="faq21">Why am I asked for a seven-day food diary?</h3>
+                        <div class="faq_content">
+                            <p>Diet is a major factor affecting the gut microbiome since our gut microbes eat the food we don’t digest. Having diet information will allow us to look for patterns in gut microbiome composition across populations of people with different diets.</p>
```
Comment (adamrp) [inline www/americangut/FAQ.psp]: Also note here that responses to all questions on the survey, including the dietary questions, are strictly voluntary.  We have had people write to the help account FURIOUS that they are asked to contribute a 7-day food log.

**Comment by adamrp** [comment]
Made one comment, but if it is addressed (or I am out-voted on that one), I'm happy with it.  Great work!

File: www/americangut/FAQ.psp (hunk 3 above)
Comment (ElDeveloper) [inline www/americangut/FAQ.psp]: :+1: this has been a delicate matter for a lot of participants.

Comment (RNAer) [inline www/americangut/FAQ.psp]: I agree. updated accordingly.

**Comment by ElDeveloper** [comment]
Ok, gonna merge this then.

**Pull request merged**